src/services/color.ts

```diff
@@ -7,10 +8,25 @@ export const lineColors: Record<LineNumber, String> = {
   4: 'purple',
 }
 
-export function getColor(
+export function getColorLine(
   colorType: String,
   lineNumber: LineNumber,
   variant: Number
 ): string {
   return `${colorType}-${lineColors[lineNumber]}-${variant}`
 }
+
+export const busColors: Record<BusNumber, String> = {
+  C1: 'lime-500', //#95C11E
```

Comment:
This is just my personal preference, but could you add space after `//` for the comment? I just realized it's not being picked in the linter. *perhaps I am influenced by [Python](https://peps.python.org/pep-0008/#inline-comments). Since we already did that in this project, is it ok if we continue to add a space after `//`?